operator/templates/repair-job.yaml

```diff
@@ -12,7 +12,21 @@ spec:
     spec:
       containers:
         - name: repair-job
-          image: bitnami/kubectl:1.18.0
-          command: [ "kubectl", "exec", "{{ $.Params.REPAIR_POD }}", "--", "nodetool", "repair" ]
+          image: bitnami/kubectl:latest
```

Comment:
I'm not sure using `latest` here is a good idea. We might want to make this a fixed version, otherwise we probably run into issues when kubectl 1.19, 1.20, etc is release

Reply:
agree on not using `latest`

Reply:
I'm changed from `1.18.0` to `latest` to be consistent with us using the same image for the node resolver init container. I agree that using `latest` is not good but neither is `1.18.0` because both are assuming a specific version of the cluster. Let me fix that here and in the `Statefulset` by introducing a new parameter to set the version of this image -- the default would be `1.18`.